enclave-runner/src/loader.rs

```diff
@@ -200,6 +205,10 @@ impl<'a> EnclaveBuilder<'a> {
         self
     }
 
+    pub fn usercall_ext<T: Into<Box<UsercallExtension>>>(&mut self, extension: T) {
```

Comment:
Expand abbreviation to `usercall_extension` (only here, and in the documentation link URL for UsercallExtension).